package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "abxtracted",
-  "version": "0.0.2",
+  "version": "0.0.4",
```

Comment:
An `v0.0.3` tag had been already created, but `package.json` was still in `0.0.2`